modules/ROOT/pages/release-notes/release-notes502.adoc

```diff
@@ -12,17 +12,17 @@ TinyMCE 5.0.2 release adds improvements to the TinyMCE editor to improve the ove
 
 === Toolbar Drawers
 
-TinyMCE 5.0.2 introduces the ability to display overflowing toolbar buttons in an extra toolbar called a "toolbar drawer". The toolbar drawer is initially hidden, and can be shown by pressing the `toolbar_drawer` icon image:{baseurl}/images/icons/more-drawer.svg[Drawer]. The button only appears if the toolbar buttons do not fit horizontally. This feature is configured using the `toolbar_drawer` configuration setting.
-Refer to the link:{baseurl}/configure/editor-appearance/#toolbar_drawer[toolbar_drawer] documentation for more information.
+TinyMCE 5.0.2 introduces the ability to display overflowing toolbar buttons in an extra toolbar called a "toolbar drawer". The toolbar drawer is initially hidden, and can be shown by pressing the `toolbar_drawer` icon image:icons/more-drawer.svg[Drawer]. The button only appears if the toolbar buttons do not fit horizontally. This feature is configured using the `toolbar_drawer` configuration setting.
+Refer to the xref:configure/editor-appearance.adoc#toolbar_drawer[toolbar_drawer] documentation for more information.
 
 === Checklist
 
 The new TinyMCE 5.0.2 editor has the ability to add checklists to the content. The *Checklist* plugin helps the user keep track of all required actions by creating a simple, clean, and easy to use TODO list.
 
-For more information on *Checklist* refer to the full link:{baseurl}/plugins/premium/checklist/[documentation].
+For more information on *Checklist* refer to the full xref:plugins/premium/checklist.adoc[documentation].
 
-There is also a demo provided to explore the *Checklist* capabilities link:{baseurl}/demo/checklist/[here].
+There is also a demo provided to explore the *Checklist* capabilities xref:demo/checklist.adoc[here].
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.2 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version502march52019[Changelog].
+The new TinyMCE 5.0.2 editor comes with significant changes to the previous versions. Tiny has created a more configurable and modern editor. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version502march52019[Changelog].
```

Comment:
This xref doesn't resolve as the `version502march52019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version502march52019]]
== 5.0.2 - 2019-03-05
```

Reply:
done